spec/support/schemas/coupon.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
what is the purpose of this schema?

Reply:
I use it for tests to make sure endpoints are returning the valid JSONs with all required attributes (defined with JSON::Serializer). Basically, I just compare the responses’ bodies with this schema, so if some field in the response is missing, the test would fail and would acknowledge missing attributes.

Reply:
let's do it with ruby